app/views/govuk_publishing_components/components/docs/warning_text.yml

```diff
@@ -17,3 +17,7 @@ examples:
     data:
       text_icon: "£"
       text: "You can be fined up to £5,000 if you don’t register."
+  without_icon:
+    data:
+      text_icon: ""
```

Comment:
I wonder if it would make more sense to be passing `text_icon: false`?

Reply:
I tried that originally, but I wanted to leave the default behaviour as it is, which is if nothing is passed the icon is shown as `!`. So if you passed false, `text_icon ||= '!'` defaults back to `!`. This seemed the cleanest way of doing it without having to update any apps.

Reply:
Chatted on Slack and decided this may be the safest option in case people have been passing in `text_icon: false` and expecting to get the default `!`